graphql2/graphqlapp/mutation.go

```diff
@@ -29,6 +32,39 @@ func (a *Mutation) SetFavorite(ctx context.Context, input graphql2.SetFavoriteIn
 	}
 	return true, nil
 }
+
+type safeErr struct{ error }
+
+func (safeErr) ClientError() bool { return true }
+
+func (a *Mutation) DebugSendSms(ctx context.Context, input graphql2.DebugSendSMSInput) (*graphql2.DebugSendSMSInfo, error) {
+	err := permission.LimitCheckAny(ctx, permission.Admin)
+	if err != nil {
+		return nil, err
+	}
+
+	err = validate.Many(
+		validate.Phone("To", input.To),
+		validate.Phone("From", input.From),
+		validate.Text("Body", input.Body, 1, 1000),
```

Comment:
Should we limit this to 255?

Reply:
Since it's an admin-only tool and not being displayed in a list or stored anywhere I left it a bit long so there's a bit more flexibility in what you want to test (e.g. multi-segment SMS). The 255 elsewhere was just for DB & title size reasons.

If we wanted to limit it I'd say we put the logic to limit it to a single segment, but if anything we could extend it in the future to show a calculated segment counter.